src/FormCollection/README.md

```diff
@@ -0,0 +1,190 @@
+# UX Form Collection
+
+Symfony UX Form collection is a Symfony bundle providing light UX for collection
+in Symfony Forms.
+
+## Installation
+
+UX Form Collection requires PHP 7.2+ and Symfony 4.4+.
+
+Install this bundle using Composer and Symfony Flex:
+
+```sh
+composer require symfony/ux-form-collection
+
+# Don't forget to install the JavaScript dependencies as well and compile
+yarn install --force
+yarn encore dev
+```
+
+Also make sure you have at least version 2.0 of [@symfony/stimulus-bridge](https://github.com/symfony/stimulus-bridge)
+in your `package.json` file.
+
+## Use predefined theme
+
+You need to select the right theme from the one you are using:
+
+```yaml
+# config/packages/twig.yaml
+twig:
+    # For bootstrap for example
+    form_themes: ['@FormCollection/form_theme_div.html.twig']
```

Comment:
I think we should only have one form theme (`form_theme_div.html.twig`). If we DID have a 2nd form theme, I would make it a bootstrap theme so that the user could instantly get something nice-looking inside of Bootstrap... but I'm not sure that's necessary because it's really easy to add `btn btn-primary` classes to the buttons from within the form.

Anyways, if the user needs to heavily customize their buttons, let's let them create their own form theme to override our's. Also, I think we should prevent the user from needing to do this step at all. We can automatically add this form theme for them: https://github.com/symfony/ux/blob/09c9effcc2582d4de0907c9e12c35afa313433ae/src/Dropzone/DependencyInjection/DropzoneExtension.php#L27-L37 (don't forget the `PrependExtensionInterface`.